### PR TITLE
sourcemap-upload: parameterize API server URL

### PR DESCRIPTION
**Pull request by jazzdan**
Fixes BAC-2792

### CODE REVIEW

File: packages/sourcemap-upload/src/index.ts [hunk 1]
```diff
@@ -203,14 +203,18 @@ async function processSourceMaps(opts: NormalizedOptions) {
   );
 }
 
+function getApiServer() {
+  return process.env.RECORD_REPLAY_API_SERVER || "https://api.replay.io/";
```
Comment (jazzdan) [inline packages/sourcemap-upload/src/index.ts:207]: We should probably standardize this environment variables somewhere.

File: packages/sourcemap-upload/src/index.ts [hunk 2]
```diff
@@ -203,14 +203,18 @@ async function processSourceMaps(opts: NormalizedOptions) {
...
+}
+
 type PutOptions = {
   groupName: string;
   apiKey: string;
   map: SourceMapToUpload;
 };
 
 async function sendUploadPUT(opts: PutOptions): Promise<Response> {
-  return fetch("https://api.replay.io/v1/sourcemap-upload", {
```
Comment (ryanjduffy) [inline packages/sourcemap-upload/src/index.ts:213]: I think we still need the `v1/`, right?

Comment (jazzdan) [reply packages/sourcemap-upload/src/index.ts:213]: Yup my bad

**Review by ryanjduffy — APPROVED** [review]

**Comment by ryanjduffy** [comment]
@jazzdan - we should add this parameter to the CLI too, I think

**Comment by ryanjduffy** [comment]
I suppose that could be a follow up but it'd be nice to not have to use a environment variable for it.

**Comment by jazzdan** [comment]
I agree with that @ryanjduffy, what do you think of this?

**Comment by ryanjduffy** [comment]
Let's also add the option to the CLI too:

https://github.com/replayio/replay-cli/blob/59dfcaf950662c7195cac94df0c29c66cd71f5f3/packages/replay/src/bin.ts#L98-L116

which will let us use `npx @replayio/replay upload-sourcemaps --group my-group --server <api-server-url>`. Leaning towards using `server` for the flag to match other commands that have that option.

wdyt?

**Comment by jazzdan** [comment]
@ryanjduffy I'm fine with that. I don't think normal users of this package will ever need to change this so I don't think it needs to be advertised per se, but for the sake of standardization I will add it there.

**Comment by ryanjduffy** [comment]
I expect you're right but it's nice for us when these configs are predictable too :)

**Pull request merged**